org/ros/rosjava_messages/tango_ros_messages/1.0.0/tango_ros_messages-1.0.0.pom

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.ros.rosjava_messages</groupId>
+  <artifactId>tango_ros_messages</artifactId>
+  <version>1.0.0</version>
+  <dependencies>
+    <dependency>
+      <groupId>org.ros.rosjava_bootstrap</groupId>
+      <artifactId>message_generation</artifactId>
+      <version>[0.2,0.3)</version>
```

Comment:
@PerrineAguiar this is pulling in message_generation from indigo, but we're currently in kinetic (0.3). Are you sure you specifically want an old version of message_generation? This may cause build dependency conflicts.
If you could please update this to `[0.3,0.4)` that will make life easier.